src/hooks/useChecked.ts

```
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+
+const useChecked = () => {
+  const [checkedList, setCheckedList] = useState<Number[]>([]);
```

Biome

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)